src/components/version-context-switcher/index.tsx

```diff
@@ -58,30 +42,15 @@ const VersionContextSwitcher = ({
 		}
 	}
 
-	/**
-	 * TODO: hopefully this is temporary. Because Sentinel does not have a
-	 * `ProductIcon` and the component intentionally returns `null` for if the
-	 * value for the `product` prop is `'sentinel'`, opting to pass a different
-	 * value for Sentinel here rather than affect all current references to
-	 * ProductIcon.
-	 *  - `ProductSwitcher` is one example that would be imacted by adding a
-	 *    ProductIcon for Sentinel. We currently do not want to render an icon
-	 *    for Sentinel in `ProductSwitcher`.
-	 */
-	const productIconSlug =
-		currentProduct.slug === 'sentinel' ? 'hcp' : currentProduct.slug
```

Comment:
We will be supporting sentinel soon, so might be good to leave this here? Or any particular reason why you want to remove it with this work?

Reply:
agree- I don't think there's any reason to remove this.

Reply:
I'm missing context here. Why would we need to keep this if we're getting rid of the product icon in the version dropdown?

Reply:
@kendallstrautman @kellyschummer ☝️

Reply:
Right! sorry I was wrong 👍🏻